api/envoy/data/tap/v2alpha/wrapper.proto

```diff
@@ -16,6 +18,17 @@ option java_package = "io.envoyproxy.envoy.data.tap.v2alpha";
 // Wrapper for all fully buffered and streamed tap traces that Envoy emits. This is required for
 // sending traces over gRPC APIs or more easily persisting binary messages to files.
 message TraceWrapper {
+  message Destination {
+    string cluster_name = 1;
+    google.protobuf.Struct cluster_metadata = 2;
```

Comment:
Docs? How are these set?

Reply:
i added more callbacks to `HttpPerRequestTapper` to support this. cluster_metadata is all the meatadata attached to the cluster under the filter's name